src/css/profile/wearable/changeable/theme-circle/arclistview.less

```diff
@@ -103,6 +103,10 @@
 					}
 				}
 			}
+			.ui-arc-listview-text-content {
+				overflow: hidden;
+				text-overflow: ellipsis;
```

Comment:
This causes problems when marquee is defined by user (As in UIComponents/list/MarqueeList oe SnapMarqueeList)
![image](https://user-images.githubusercontent.com/38784474/62038593-535a2a80-b1f6-11e9-8554-17e21a4649af.png)